### PR TITLE
New Emag Shuttle: The Suspicious Shuttle

### PR DESCRIPTION
**Pull request by neeshacark**
# Document the changes in your pull request

Adds a new shuttle. 
image

Comes with 12 suspicious, sentient, and murderous goats, plenty of vents, sizeable departments, and an abandoned cargo area. This particular shuttle was a strange artefact that fell through bluespace into the Nanotrasen shipyards, its design is forbidden. As such, it is only available via emag.

i am not sorry

# Why is this good for the game?
more shuttle

# Changelog

:cl: cark
mapping: adds the suspicious shuttle
/:cl:

### CODE REVIEW

**Comment by Beeper-Skeeper** [comment]
"Red sus"
"WE'RE ALL RED YOU FUCKING IDIOT"

**Review by ynot01 — APPROVED** [review]

File: code/datums/shuttles/emergency.dm [hunk 1]
```diff
@@ -127,6 +127,14 @@
 	credit_cost = 5000 // While the shuttle is rusted and poorly maintained, trained bears are costly.
 	emag_only = TRUE
 
+/datum/map_template/shuttle/emergency/amongushuttle
+	suffix = "amongushuttle"
```
Comment (ynot01) [inline code/datums/shuttles/emergency.dm]: filename is "emergency_amogushuttle"

**Review by ynot01 — CHANGES_REQUESTED** [review]

**Comment by neeshacark** [comment]
i have no idea why it's not compiling

**Comment by ynot01** [comment]
> i have no idea why it's not compiling

```
[05:04:54] Runtime in _shuttles.dm,49: cannot read from list
  proc name: preload size (/datum/map_template/shuttle/preload_size)
  src: Suspicious Shuttle (/datum/map_template/shuttle/emergency/amogushuttle)
  call stack:
  Suspicious Shuttle (/datum/map_template/shuttle/emergency/amogushuttle): preload size("_maps/shuttles/emergency_amogu...", 0)
  Suspicious Shuttle (/datum/map_template/shuttle/emergency/amogushuttle): New(null, null, 0)
  Suspicious Shuttle (/datum/map_template/shuttle/emergency/amogushuttle): New()
  Suspicious Shuttle (/datum/map_template/shuttle/emergency/amogushuttle): New()
  Mapping (/datum/controller/subsystem/mapping): preloadShuttleTemplates()
  Mapping (/datum/controller/subsystem/mapping): preloadTemplates("_maps/templates/")
  Mapping (/datum/controller/subsystem/mapping): Initialize(null)
  Master (/datum/controller/master): init subsystem(Mapping (/datum/controller/subsystem/mapping))
  Master (/datum/controller/master): Initialize(10, 0, 1)
```

**Comment by neeshacark** [comment]
i do not know what this means

**Review by ynot01 — APPROVED** [review]

**Review by AMyriad — CHANGES_REQUESTED** [review]
1. Take away outer layer of walls please, don't believe they're needed for this
2. Take away the 2 extra dir-windows around the abandoned area please. If you want em to block sight or be tougher just varedit it
3. Can we get some dir-windows and/or firelocks inbetween the shuttle chairs in the middle? Space is a bit open and could use some more geometry or cover
4. Would like to see command/sec/med areas be a little more easier to identify by having some signs or use command/sec doors, as it stands it all blends together a bit much

**Review by AMyriad — APPROVED** [review]

**Pull request merged**